packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -26,9 +26,12 @@ module.exports = (resolve, rootDir, isEjecting) => {
     collectCoverageFrom: ['src/**/*.{js,jsx}'],
     setupFiles: [resolve('config/polyfills.js')],
     setupTestFrameworkScriptFile: setupTestsFile,
-    testMatch: [
-      '<rootDir>/src/**/__tests__/**/*.js?(x)',
-      '<rootDir>/src/**/?(*.)(spec|test).js?(x)',
+    // the `testMatches` option currently watches node_modules
+    // Which causes crashes on large projects on MacOS.
+    // Once this is fixes we can change from `modulePathIgnorePatterns` to `testMatches`.
+    // See facebook/jest/issues/1767 & facebookincubator/create-react-app/pull/2395
+    modulePathIgnorePatterns: [
+      '<rootDir>[/\\\\](build|dist|docs|node_modules|coverage|scripts|bower_components)[/\\\\]',
```

Comment:
Seems like this still includes any arbitrary folders other than these? We specifically fixed it in 1.0 and I don't want to regress on it. Can we use a negative lookahead to ignore everything but `src`?